src/components/SideBar/SideBarItem.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Icon, Text, Flex } from "@chakra-ui/react";
+
+const SideBarItem = (props: any) => {
+  return (
+    <a data-testid={props.item_text}>
+      <Box w="100%" p="32px 7px">
+        <Flex flexDirection="column" align="center" justify="flex-start">
+          <Icon
+            viewBox={props.item_viewBox}
+            fill="none"
+            boxSize="28px"
+            xmlns="http://www.w3.org/2000/svg"
+            color="#C8C5C5"
+          >
+            {props.children}
+          </Icon>
+          <Text
+            m="8px"
+            as="span"
+            fontSize="16px"
+            fontWeight="700"
+            lineHeight="19.4px"
```

Comment:
Chakra has default theme, you can use it
<img width="1589" alt="image" src="https://user-images.githubusercontent.com/61776449/168309203-ff21cf84-6a10-4b63-8bee-83d2509610a6.png">

In addition, the value here is different from figma.
<img width="1458" alt="image" src="https://user-images.githubusercontent.com/61776449/168309400-538d9c7f-36e3-429d-8cea-3bb37a7cbf03.png">